packages/vm/src/modules/in_memory_cache.rs

```diff
@@ -1,13 +1,13 @@
 use clru::CLruCache;
-use wasmer::{Module, Store};
+use wasmer::Module;
 
 use crate::{Checksum, Size, VmResult};
 
 const ESTIMATED_MODULE_SIZE: Size = Size::mebi(10);
 
 /// An in-memory module cache
 pub struct InMemoryCache {
-    artifacts: CLruCache<Checksum, Vec<u8>>,
+    artifacts: CLruCache<Checksum, Module>,
```

Comment:
All changes in this file are good. I'd just rename this field to `modules`, since we don't operate on artifact level anymore.